Prog/average_unitsize_sf_mf.sas

```diff
@@ -0,0 +1,59 @@
+/**************************************************************************
+ Program:  average_unitsize_sf_mf.sas
+ Library:  RegHsg
+ Project:  NeighborhoodInfo DC
+ Author:   Yipeng Su
+ Created:  2/6/19
+ Version:  SAS 9.2
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  tabulate sf and mf average unit size for DC MSA
+ codebook: https://www.census.gov/data-tools/demo/codebook/ahs/ahsdict.html?s_appName=ahsdict&s_searchvalue=INTSTATUS
+ Modifications: 
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( RegHsg)
+
+/*
```

Comment:
if you fix the issue of where the formats are stored - i don't think you need these?